deploy.py

```diff
@@ -34,134 +37,154 @@ def deploy(bento_bundle_path, deployment_name, config_json):
-            "--deployment-container-image-name",
-            docker_image_tag,
-            "--docker-registry-server-user",
-            docker_username,
-            "--docker-registry-server-password",
-            docker_password,
-        ]
-    )
+    with console.status("Pushing image"):
+        build_docker_image(
+            context_path=deployable_path,
+            image_tag=docker_image_tag,
+            dockerfile="Dockerfile-azure",
+            additional_build_args={
+                "BENTOML_VERSION": LAST_PYPI_RELEASE_VERSION,
+                "PYTHON_VERSION": major + minor,
+            },
+        )
+        push_docker_image_to_repository(docker_image_tag)
+    console.print(f"Pushed image {docker_image_tag}")
+
+    with console.status("Deploying image in Azure functions"):
+        docker_username, docker_password = get_docker_login_info(
+            resource_group_name, acr_name
+        )
+        run_shell_command(
+            [
+                "az",
+                "functionapp",
+                "create",
+                "--name",
+                function_name,
+                "--storage-account",
+                storage_account_name,
+                "--resource-group",
+                resource_group_name,
+                "--plan",
+                function_plan_name,
+                "--functions-version",
+                "3",
+                "--deployment-container-image-name",
+                docker_image_tag,
+                "--docker-registry-server-user",
+                docker_username,
+                "--docker-registry-server-password",
+                docker_password,
+            ]
+        )
+    console.print(f"Deployed in Azure function {function_name}")
     set_cors_settings(function_name, resource_group_name)
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 4:
-        raise Exception(
-            "Please provide bento_bundle_path deployment_name and configuration json"
-        )
-    bento_bundle_path = sys.argv[1]
-    deployment_name = sys.argv[2]
-    config_json = sys.argv[3] if sys.argv[3] else "azure_config.json"
-
-    deploy(bento_bundle_path, deployment_name, config_json)
+    parser = argparse.ArgumentParser(
+        prog='deploy',
+        description="Deploy the bentoml bundle on Azure Functions",
+        epilog="Check out https://github.com/bentoml/azure-functions-deploy/blob/main/README.md to know more",
+    )
+    parser.add_argument("bento_bundle_path", help="Path to bentoml bundle")
+    parser.add_argument(
+        "deployment_name", help="The name you want to use for your deployment"
+    )
+    parser.add_argument(
+        "config_json",
+        help="(optional) The config file for your deployment",
+        default=os.path.join(os.getcwd(), "azure_config.json"),
+        nargs="?",
+    )
+    args = parser.parse_args()
+    deploy(args.bento_bundle_path, args.deployment_name, args.config_json)
+    pprint("Deployment complete!")
```

Comment:
can you change pprint to console.print. pprint is used to display a python object but what we want here is a console output.

Reply:
Changed pprint("Deployment complete!") to console.print("Deployment complete!") in deploy.py and have commited the change